tests/test_posixDatastoreFits.py

```diff
@@ -132,10 +132,10 @@ def testTransfer(self):
         ref = self.makeDatasetRef("calexp", dataUnits, storageClass, dataId)
 
         inputConfig = DatastoreConfig(self.configFile)
-        inputConfig['datastore.root'] = os.path.join(self.testDir, "./test_input_datastore")
+        inputConfig['root'] = os.path.join(self.testDir, "./test_input_datastore")
         inputPosixDatastore = PosixDatastore(config=inputConfig, registry=self.registry)
         outputConfig = inputConfig.copy()
-        outputConfig['datastore.root'] = os.path.join(self.testDir, "./test_output_datastore")
+        outputConfig['root'] = os.path.join(self.testDir, "./test_output_datastore")
```

Comment:
Are we now assuming that `Datastore` and `Registry` always have the same `root`? What if (for a particular datastore) there is no such thing?

Reply:
I'm not sure I understand. This is a `DatastoreConfig` so by definition it's a datastore config not a registry config. Or are you asking a different question?